### PR TITLE
Add a new packet read count test

### PR DESCRIPTION
**Pull request by antoineMoPa**
Here is this test failing without the fix:

image

This edge case was pretty frequent, since we often seek to 0, then seek to a different time.

Before this morning's fix (RE_SEEK_THRESHOLD), we would read all packets until the target (501 packets in the screenshot!). With RE_SEEK_THRESHOLD, we skip to a closer position when appropriate.

### CODE REVIEW

File: test/framefusion.test.ts [hunk 1]
```diff
@@ -89,8 +89,32 @@ describe('FrameFusion', () => {
...
+        const extractor = await BeamcoderExtractor.create({
+            inputFileOrUrl: 'https://storage.googleapis.com/lumen5-prod-video/mvc-4k-new-orleans-a053c0340725rv-112014WA74Rf.mp4',
+        });
+
+        // Act & assert
+        await extractor.getFrameAtTime(0);
+        expect(extractor.packetReadCount).to.be.below(15);
+
+        await extractor.getFrameAtTime(20);
+        expect(extractor.packetReadCount).to.be.below(15);
```
Comment (antoineMoPa) [inline test/framefusion.test.ts:108]: I love that we have `packetReadCount`. It will allows a bunch of performance tests, without having to rely on timing.

Maybe we should make it private though?

**Review by animanathome — APPROVED** [review]
Thank you. Now the code makes a bit more sense.
